playbooks/02 - Use Case - ICS Advisory and CVEs/CVE Changes To KEV.json

```diff
@@ -4,7 +4,7 @@
     "name": "CVE Changes To KEV",
```

Comment:
@cs-ankit-gupta as discussed, add relevant playbook trigger tags to the playbooks